touchin-custom-view/Code/View/view.swift.liquid

```diff
@@ -0,0 +1,42 @@
+import UIKit
+import LeadKit
+
+final class {{ module_info.name }}View: BaseView {
```

Comment:
BaseInitializeableView

Reply:
I can't find it in scope) 

BaseView is a subclass of UIVIew, implementing InitializableView protocol

Reply:
as well as BaseView
none of them in LeadKit right now